### DISCUSSION TITLE
OpenAI & ChatGPT 如何判断流量是否经过了 v2ray

### DISCUSSION THREAD

**Discussion by ZXiangQAQ** [v2fly/v2ray-core#2614] [Q&A]
使用的整体方式为 v2ray + websocket + NG + SSL

用来很久一切都正常，昨天突然无法访问 Chatgpt，提示 Unable to load site，Please try again later. If you are using a VPN, try turning it off. Check the [status page](https://status.openai.com/) for information on outages.

ChatGPT 是如何判断我使用了 v2ray 的，直接封的IP，还是封的网段，还是对我的流量做了嗅探或者特征识别，有了解的大佬可以帮助解答下么

**Comment by nursery01** [comment]
gpt封了部分IP區段，比如甲骨文的

**Comment by petulat** [comment]
我也是，突然出现了这个提示，请问你解决了吗？是设置问题，还是IP问题？

**Reply by ZXiangQAQ** [reply]
目前看是IP问题，我换了个机房，怕同机房网段都被干了

**Comment by tyreverzore** [comment]
我美国的机房的vps都被封了

**Comment by erlou96** [comment]
同样，HK 和 US 的两个小鸡都被ban了
